index.js

```diff
@@ -1,14 +1,13 @@
 const a11y = require.resolve('./rules/a11y');
 const vue = require.resolve('./rules/vue');
 const vuex = require.resolve('./rules/vuex');
-const testId = require.resolve('./rules/test-id');
 
 module.exports = {
-  extends: ['plugin:vue/recommended', a11y, vue, vuex, testId],
+  extends: ['plugin:vue/recommended', a11y, vue, vuex, 'plugin:test-id/recommended'],
   parserOptions: {
     parser: 'babel-eslint',
   },
-  plugins: ['vue', 'vuejs-accessibility', 'test-id'],
+  plugins: ['vue', 'vuejs-accessibility'],
```

Comment:
Thanks for adding extends support, @rwpswami 🎉 I think we still want to list the plugin in addition to the extends section. Would you mind adding it back?

```suggestion
  plugins: ['vue', 'vuejs-accessibility', 'test-id'],
```

Reply:
Yes sure. Added back.

Reply:
@tclindner Please take a look